src/commons/stores/cluster-store.js

```diff
@@ -118,6 +118,25 @@
                 temp.hosts = store.getAssociatedHosts(data[i]);
                 res.push(temp);
             }
+
+            function _getProfileStatus(status) {
```

Comment:
Cleaner way to write:

profileState[volume_profiling_state]

var profileState = {"enable": "Enabled",
"disabled": "Disabled",
"mixed": "Mixed"}